### DISCUSSION TITLE
Not able start the workstation

### DISCUSSION THREAD

**Discussion by Akhileshreddy9181** [learndevopsonline/learndevopsonline#424] [Q&A]
![image](https://user-images.githubusercontent.com/55424105/230162181-82a82d00-4d3c-49bd-b032-5ee778b118d3.png)

**Comment by noolivara** [comment]
@Akhileshreddy9181 , Can you please click on spot requests and check the status of the Spot Instance request. 
Can you please attach screenshot.

**Reply by Akhileshreddy9181** [reply]
Got recovered. It's working fine now.
